Public/Src/Cache/ContentStore/Library/FileSystem/TrackingFileStream.cs

```diff
@@ -32,28 +32,28 @@ private string Path
         }
 
         /// <inheritdoc />
-        public TrackingFileStream([NotNull] string path, FileMode mode)
+        public TrackingFileStream([JetBrains.Annotations.NotNull] string path, FileMode mode)

```

Comment:
>[JetBrains.Annotations. [](start = 34, length = 23)

wow, this is pretty annoying. What is this clashing with? I didn't see a NotNull in NetCore 3.0. Maybe I'm not looking hard enough

Reply:
If we have some contention on this type name, we need to do something less intrusive. Let's chat when you get a chance.

---
In reply to: [326408729](https://github.com/microsoft/BuildXL/pull/903#discussion_r326408729) [](ancestors = 326408729)